src/main/java6/net/finmath/marketdata/model/curves/AbstractForwardCurve.java

```diff
@@ -145,4 +119,24 @@ public double getPaymentOffset(double fixingTime) {
 
 		return values;
 	}
+	
+	@Override
+	public double getForward(double fixingTime)
+	{
+		return this.getForward(null, fixingTime, getPaymentOffset(fixingTime));
+	}
+	
+	@Override
+	public String getDiscountCurveName() {
```

Comment:
Why have a getDiscountCurveName method, if there is no longer the member discountCurveName?